addon/templates/components/frost-modal-dialog.hbs

```diff
@@ -1,44 +1,64 @@
-<div class="overlay">
-  <div class="modal">
+{{#ember-remodal
+  forService=true
+  closeOnOutsideClick=false
+  onOpen=(action 'modalOpen')
+  onClose=(action 'modalClose')
+  onConfirm=(action 'confirm')
+  closeOnEscape=false
+  disableNativeClose=true as |modal|
+}}
+  {{yield}}
+  {{#modal.open}}
```

Comment:
Is the intent to always have {{modal.open}} show or would you want this to only show if the block-slot 'target' is used? So should the {{modal.open}} be wrapped inside the {{yield-slot 'target'}} ?